tests/integration_tests/tests/agentless_tests/test_plugin_labels.py

```diff
@@ -123,12 +126,32 @@ def test_resource_tags_propagation(self):
                {
                    'key1': 'value1',
                    'key2': 'value2',
+                   'boolean_value': False,
+                   'deployment_id': {'get_sys': ['deployment', 'id']},
+                   'owner': {'get_sys': ['deployment', 'owner']},
                }
 
         self.client.deployments.delete('dep')
         self._wait_until_deployment_deleted('dep')
         self.client.blueprints.delete('bp')
 
+    def test_resource_tags_intrinsic_functions(self):
+        self.client.blueprints.upload(
+            resource('dsl/blueprint_with_resource_tags.yaml'),
+            entity_id='bp')
+        self.client.deployments.create('bp', 'dep')
+
+        execution = self.client.executions.start(
+            deployment_id='dep',
+            workflow_id='execute_operation',
+            parameters={'operation': 'test.context'}
+        )
+
+        self.wait_for_execution_status(execution.id, 'completed')
```

Comment:
this needs to be "terminated", not "completed".  "Completed" is just the display text, but the actual value is "terminated".

Or really, just:
```suggestion
        self.wait_for_execution_to_end(execution)
```
(this already throws if the execution fails)